unidist/core/backends/mpi/core/common.py

```diff
@@ -55,6 +56,19 @@ class Operation:
     CANCEL = 11
 
 
+operations_dict = dict(
+    (value, name)
+    for name, value in inspect.getmembers(Operation)
+    if name not in dir(type("dummy", (object,), {}))
+)
```

Comment:
Set `dir(type("dummy", (object,), {}))` to a new var. Add a comment for it and for `operations_dict`. Make both of the vars private.